kata.py

```diff
@@ -0,0 +1,69 @@
+#Imports
+import json
+
+class Parcel:
+    def __init__(self):
+        self.name=""
+        self.dimension=0
+        self.size=""
+        self.cost=0
+        self.weight=0
+
+    def create_parcel(self, myName, myDimension, myWeight):
+        self.name=myName
+        self.dimension=myDimension
+        self.weight=myWeight
+        self.get_size()
+        self.cost=self.sizeTocost(self.size)
+        self.addOverweight()
+
+    def get_size(self):
+        if(self.weight>50):
+            self.size="Heavy"
```

Comment:
Slight confusion of domain concepts, `Heavy` isn't really a size, also it can sometimes be more efficient to make a lighter parcel heavy for cost savings